packages/ddex/entrypoint/README.md

```diff
@@ -0,0 +1,48 @@
+# @audius/sdk + React + TypeScript + Vite
```

Comment:
can this readme look more like [the webapp one](https://github.com/AudiusProject/audius-protocol/blob/main/packages/ddex/webapp/README.md) with a quick summary, command to run with hot reloading, and link to the packages/ddex/README.md?

Reply:
yes absolutely